docs/ballerina-by-example/examples/fork-join-variable-access/fork-join-variable-access.description

```diff
@@ -1,8 +1,9 @@
-//Workers within fork-join can access any variables which are declared within
-//the scope of that statement.
-//Value type variables are passed into these workers as a copy of the original
-//data.
-//Reference type variables are passed into these workers as a reference to the
-//original data.
-//Join block receives a map of any type array which contains the data sent from
-//the respective workers.
+// Workers within a fork-join can access the variables that are declared within the scope of that statement.
+// The variables can be value type variables or reference type variables.
+// Value type variables, such as string, integer, float, and blob type variables, are passed into the workers as a copy
+// of the original data. Therefore, you are able to update the value of it within the worker and that value is only
+// defined within the specific worker.
+// Reference type variables, such as map, json, xml, and records type variables, are passed into these workers as a
+// reference to the original data. Therefore, if you update the value of these variables within a worker, the value gets
+// updated for the entire function.
+// The Join block receives a results from the workers into a map that is of of the 'any' type array.
```

Comment:
can we change this statement (line 9). It should be map of any type array. Also there is two "of"